### PR TITLE
PE-388 Add dismissible functionality to alerts

### PR DESCRIPTION
**Pull request by radmosley**

### CODE REVIEW

File: web/sites/default/config/views.view.alerts_page.yml [hunk 1]
```diff
@@ -0,0 +1,204 @@
+uuid: 2cd8ec86-f937-47d6-baf0-9d382888b24a
```
Comment (RichardDavies) [inline web/sites/default/config/views.view.alerts_page.yml:1]: Why did you move this from the `alert` view into a completely separate view? We should be trying to consolidate our views whenever possible and I don't see any reason why this needs to be a separate view.

Comment (radmosley) [reply web/sites/default/config/views.view.alerts_page.yml:1]: I  used a different view in order to apply a different display mode to the alerts that appear on the alerts page versus globally on the site. This allows for alerts to appear on the alerts page without the dismissible button.

**Review by RichardDavies — APPROVED** [review]
